package.json

```diff
@@ -39,9 +39,11 @@
     "nodemailer": "1.x.x",
     "nodemailer-markdown": "1.x.x",
     "object-assign": "4.x.x",
-    "react": "0.14.x",
+    "react": "^0.14.x",
+    "react-dom": "^0.14.8",
```

Comment:
Please follow the pattern of no `^` character and using `x` after largest versions. In this case `0.14.x`, if they have major versions, it should be `N.x.x`.